packages/react-next/src/components/Checkbox/examples/Checkbox.Basic.Example.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
I believe examples should go to `packages/examples` package. @ecraig12345  can you confirm?

Reply:
This is copied directly from the OUFR package